omero/developers/Web/ViewPort.txt

```diff
@@ -40,6 +41,22 @@ Then in ``<BODY>`` insert the following:
 
         <a href="#" onclick="return openPopup('http://localhost:8000/webclient/img_detail/IMAGE_ID/')">Open viewer</a>
 
+
+Example: 
+^^^^^^^^
+
+.. raw:: html
```

Comment:
This just doesn't appear in the PDF at all, so we get "Example:" with nothing below. Note that Sphinx allows one to make text conditional on if it's for the HTML or the PDF version, so you could probably still find a way to keep this but also to render the HTML or to omit the heading altogether for the PDF version.